src/cli/cli_coap_secure.cpp

```diff
@@ -207,7 +207,7 @@ template <> otError CoapSecure::Process<Cmd("set")>(Arg aArgs[])
  *     `check-peer-cert` is `true`, and the `max-conn-attempts` value is the
  *     number specified in the argument.
  * @par
- * Starts the CoAP Secure service.
+ * Starts the CoAP Secure service. @moreinfo{@coaps}.
```

Comment:
The `@moreinfo` tag is only added a subset of the `coaps` commands. What about other commands like `coaps get` and `coaps put`?

Reply:
@jwhui - Hi Jonathan, sure I can add the tag to more of the commands.  Will do so now.  Thanks.